### PR TITLE
:bullettrain_front: first round of frontend changes

### PR DESCRIPTION
**Pull request by ajthomascouk**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-31.dev.c2s.nhschoices.net](http://profiles-pr-31.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-31.dev.c2s.nhschoices.net](http://profiles-pr-31.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-31.dev.c2s.nhschoices.net](http://profiles-pr-31.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-31.dev.c2s.nhschoices.net](http://profiles-pr-31.dev.c2s.nhschoices.net)

File: app/views/book-appointment.nunjucks [hunk 1]
```diff
@@ -1,5 +1,95 @@
...
+
+{% block header %}
+<h1 class="local-header__title">Book a GP appointment at {{gp.name}}</h1>
+{% endblock %}
+
+{% block pageIntro %}
+<div class="local-header__intro">
+  <div class="reading-width">
+    <ul class="link-list">
+      <li><a href="#">Contact reception</a></li>
```
Comment (beseven) [inline app/views/book-appointment.nunjucks]: anchors, please

File: app/views/book-appointment.nunjucks [hunk 2]
```diff
@@ -1,5 +1,95 @@
...
+      <td>7:30am to 6pm</td>
+    </tr>
+    <tr>
+      <td>Friday</td>
+      <td>7:30am to 6pm</td>
+    </tr>
+  </table>
+  <p>You can also ask for a late appointment between 6pm and 8pm on Wednesdays and Thursdays.</p>
+  <p>The surgery has telephone appointments available, Monday to Friday 7.30am to 9am.</p>
+  <h2>Book an appointment online</h2>
```
Comment (beseven) [inline app/views/book-appointment.nunjucks]: a link, please

**Review by beseven — CHANGES_REQUESTED** [review]

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-31.dev.c2s.nhschoices.net](http://profiles-pr-31.dev.c2s.nhschoices.net)

**Review by beseven — APPROVED** [review]

**Comment by beseven** [comment]
LOTM!

**Review by neilbmclaughlin — APPROVED** [review]

**Comment by neilbmclaughlin** [comment]
Approved against 1441bcb

**Comment by neilbmclaughlin** [comment]
@ajthomascouk - rebase please

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-31.dev.c2s.nhschoices.net](http://profiles-pr-31.dev.c2s.nhschoices.net)

**Comment by ajthomascouk** [comment]
@neilbmclaughlin wrote:
> rebase please

Done... Merge when you can please.

File: scss/c2s/c2s.scss [hunk 3]
```diff
@@ -1,5 +1,8 @@
 @import "betahealth/nhsuk";
 
+$callout-border-blue: #40ADD7;
```
Comment (st3v3nhunt) [inline scss/c2s/c2s.scss]: This looks like duplication, checkout lines 10 and 11 https://github.com/nhsuk/profiles/blob/1dfdc675cfa12e5157b82c055bf66edbf232d63d/scss/c2s/c2s.scss

File: public/css/nhsuk.css [hunk 4]
```diff
@@ -897,6 +897,8 @@ form .form-group { -webkit-box-sizing: border-box; -moz-box-sizing: border-box;
 
 .link-back:link, .link-back:visited, .link-back:hover, .link-back:active { color: #000000; }
 
+.callout--blue { background-color: rgba(64, 173, 215, 0.1); border-left: 8px solid #40ADD7; padding: 8px 16px; }
```
Comment (st3v3nhunt) [inline public/css/nhsuk.css:898]: Duplicated from line 789?
https://github.com/nhsuk/profiles/blob/1dfdc675cfa12e5157b82c055bf66edbf232d63d/public/css/nhsuk.css#L789

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
Looks like some duplication has crept in

File: app/views/book-appointment.nunjucks [hunk 5]
```diff
@@ -1,5 +1,95 @@
...
+  </div>
+</div>
+{% endblock %}
+
 {% block content %}
-Book an appointment at {{gp.name}}
+<div class="reading-width">
+  <h2 id="contact-reception">Contact reception</h2>
+  <h3>Book an appointment</h3>
+  <p>Contact reception: {{gp.phone}}</p>
```
Comment (markysoft) [inline app/views/book-appointment.nunjucks]: The phone number is showing as empty on the page. In the latest schema the phone number has moved to 'contact.telephone', so needs to change if profiles 0.2.0 is being used.

**Review by markysoft — CHANGES_REQUESTED** [review]

File: app/views/book-appointment.nunjucks (hunk 5 above)
Comment (ajthomascouk) [reply app/views/book-appointment.nunjucks]: Sorted.

File: scss/c2s/c2s.scss (hunk 3 above)
Comment (ajthomascouk) [reply scss/c2s/c2s.scss]: Whoops. Sorted

File: public/css/nhsuk.css (hunk 4 above)
Comment (ajthomascouk) [reply public/css/nhsuk.css:898]: Sloppy.. Removed the dupe.

**Comment by st3v3nhunt** [comment]
@ajthomascouk Can you rebase this when you get chance please? It isn't going to be possible to merge until, thanks!

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-31.dev.c2s.nhschoices.net](http://profiles-pr-31.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-31.dev.c2s.nhschoices.net](http://profiles-pr-31.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-31.dev.c2s.nhschoices.net](http://profiles-pr-31.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-31.dev.c2s.nhschoices.net](http://profiles-pr-31.dev.c2s.nhschoices.net)

**Review by st3v3nhunt — APPROVED** [review]

**Comment by st3v3nhunt** [comment]
@markysoft Just need your review and merge  on this now please

**Review by st3v3nhunt — APPROVED** [review]
Reviewed against [42ce254](https://github.com/nhsuk/profiles/pull/31/commits/dff42ce2547b62f2d30a29b45f9ccbef54ef244a)

**Review by markysoft — APPROVED** [review]

**Pull request merged**